tests/terraform/graph/graph_builder/test_local_graph.py

```diff
@@ -108,6 +108,25 @@ def test_set_variables_values_from_modules(self):
             else:
                 self.assertEqual(var_value, default_val)
 
+    def test_compare_multi_graph_defs(self):
+        resources_dir = os.path.realpath(os.path.join(TEST_DIRNAME,
+                                                      '../resources/variable_rendering/render_module_postgresql'))
+        hcl_config_parser = TFParser()
+        module, defs = hcl_config_parser.parse_hcl_module(resources_dir, source=self.source)
+        modules = hcl_config_parser.parse_multi_graph_hcl_module(resources_dir, source=self.source)
+        for idx, module_to_definitions in enumerate(modules):
+            assert module_to_definitions[0] == module
```

Comment:
why would that be true? is it just one module always for this example?

Reply:
Yes we have a single module and then a list of TF definitions, one per file.